elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -17,12 +17,21 @@ $LOCAL-VARIABLES: (
   ::slotted(*) {
     color: #{pfe-broadcasted(text)};
   }
-
   [tabindex] {
     display: flex;
     height: 100%;
   }
 
+ //updated saturated styles
+  @at-root #{&}([pfe-variant="earth"][on="saturated"]) .container{
+    background-color: #fff;
```

These could be converted to theme colors
```suggestion
    background-color: pfe-var(surface--lightest);
```